cylc/flow/data_store_mgr.py

```diff
@@ -2017,6 +2017,16 @@ def update_family_proxies(self):
         if self.updated_state_families:
             self.state_update_follow_on = True
 
+    @staticmethod
+    def _helper(tp_delta, tp_node, label):
+        this = tp_delta
+        if this is None or not this.HasField(label):
+            this = tp_node
+        value = getattr(this, label, None)
```

Comment:
I don't think we tolerated missing attributes before. Best to keep the code brittle against renaming:

```suggestion
        value = getattr(this, label)
```